api.planx.uk/modules/flows/copyFlow/controller.ts

```diff
@@ -0,0 +1,54 @@
+import { z } from "zod";
+import { ValidatedRequestHandler } from "../../../shared/middleware/validate";
+import { Flow } from "../../../types";
+import { ServerError } from "../../../errors";
+import { copyFlow } from "./service";
+
+interface CopyFlowResponse {
+  message: string;
+  inserted: boolean;
+  replaceValue: string;
+  data: Flow["data"];
+}
+
+export const copyFlowSchema = z.object({
+  params: z.object({
+    flowId: z.string(),
+  }),
+  body: z.object({
+    replaceValue: z.string().length(5),
```

Comment:
Strictly controlling the length here to 5 matches the current frontend behaviour, and has meant that a few test cases need nodeIds tweaked below.

I'll double check the onboarding scripts to enforce this to 5 characters there as well once this is merged.